src/pipelines.py

```diff
@@ -99,9 +106,6 @@ def xgboost(config, train_mode, suffix=''):
 
 
 def sklearn_main(config, ClassifierClass, clf_name, train_mode, suffix='', normalize=False):
```

Comment:
@kstrzala let's just call it `sklearn_pipeline` or `sklearn_classifier` or something along those lines. `sklearn_main` could mean a lot of things